deploy_tools/cli.py

```diff
@@ -93,7 +93,7 @@ def validate_address(ctx, param, value):
 optimize_option = click.option(
     "--optimize",
```

Comment:
Please have a look here and define an off switch

Reply:
http://click.palletsprojects.com/en/5.x/options/#boolean-flags

Reply:
Thanks for this good hint. Should have thought more when doing this.
:heavy_check_mark: